File: submissions/Harini S/DAY 16/day16_factorial.py
num=int(input("Enter a num: "))
fact=1
if num>=0:
    if num==0 or num==1: fact=1
    else:
        for i in range(1,num+1): fact*=i
    print(f"Factorial of {num} is {fact}")
else: print("Enter a positive number.")

File: submissions/Utkarsh Saxena/day_16_factorial.c
#include <stdio.h>
int main(){
    int num, fact=1;
    printf("Enter a number: ");//taking input
    scanf("%d", &num);

    for(int i=1;i<=num;i++){
        fact *= i; //calculating factorial
    }
    printf("Factroial of %d is %d",num, fact); //printing final result
   return 0;
}